src/traces/pie/plot.js

```diff
@@ -313,12 +313,12 @@ function attachFxHandlers(sliceTop, gd, cd) {
 
     // hover state vars
     // have we drawn a hover label, so it should be cleared later
-    var hasHoverLabel = false;
+    if(!('_hasHoverLabel' in trace)) trace._hasHoverLabel = false;
```

Comment:
What about using the line below?
<pre>trace._hasHoverLabel = !!trace._hasHoverLabel;</pre>

Reply:
I don't want to cast `undefined` to `false`.